addons/website_slides/static/src/js/slides_course_fullscreen_player.js

```diff
@@ -540,15 +544,23 @@ odoo.define('website_slides.fullscreen', function (require) {
             this.sidebar.goNext();
         },
         /**
-         * Show or Hide the sidebar
+         * Click sidebar visibility toggle
```

Comment:
`Called when the sidebar toggle is clicked -> toggles the sidebar visibility` ... or no comment ;)